src/routes/game/components/dialogs/components/gameOver/components/MatchStatusBanner.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <section class="match-status-banner" :class="wrapperClass" data-cy="continue-match-banner">
+    <div class="banner-content">
+      <v-icon
+        v-if="!gameStore.opponentDeclinedRematch"
+        :icon="matchStatusIcon"
+        color="surface-2"
+        :data-cy="matchStatusIconDataCy"
+      />
+      <h2 class="banner-h2">
+        {{ headerText }}
+      </h2>
+      <v-icon
+        v-if="!gameStore.opponentDeclinedRematch"
+        :icon="matchStatusIcon"
+        color="surface-2"
+        :data-cy="matchStatusIconDataCy"
+      />
+    </div>
+  </section>
+</template>
+
+<script setup>
+import { computed } from 'vue';
+import { useGameStore } from '@/stores/game.js';
+
+const gameStore = useGameStore();
+
+const isRanked = computed(() => gameStore.isRanked);
+const isSpectating = computed(() => gameStore.isSpectating);
```

Comment:
I wonder if it makes sense to use pinias built in storeToRefs()

Reply:
https://pinia.vuejs.org/core-concepts/

Reply:
This is intriguing and something for us to look at globally